docs/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+import { styled } from "linaria/react";
+import { tm } from "../themes";
+import { DocumentationSidebarStructure } from "../config";
+
+interface Props {
+  elementsList: typeof DocumentationSidebarStructure;
```

Comment:
Let's specify this type explicitly. The reason - we will generate this structure at building time so we won't have this shape in config

Reply:
Done